news/2024/2024-09-04-osumania-4k-world-cup-2024-round-of-16.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+title: "osu!mania 4K World Cup 2024: Round of 16 Recap"
+date: 2024-09-04 21:00:00 +0000
+---
+
+The best moments of MWC4K's Round of 16 have been packaged, and delivered straight to your front page.
+
+![](/wiki/Tournaments/MWC/2024_4K/img/mwc4k2024-banner.jpg)
+
+Don't be fooled by the fact that the tournament is still in its early stages, things have already started to get heated, with many exciting matches and moments that are worthwhile of taking a look back and witnessing once more.
+
+## Mappool showcase
+
+<style>
+    .osu-md--news .audio-player {
+        max-width: inherit;
+    }
+</style>
+
+[**Download the pool here! (111 MB)**](https://osu.ppy.sh/beatmaps/packs/P258)
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2233735176&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+The mappool size has increased for this week by introducing two new slots — chordjack ([RC3](https://osu.ppy.sh/beatmapsets/2238657#mania/4757017)) and hybrid 3 ([HB3](https://osu.ppy.sh/beatmapsets/2238644#mania/4756987)). Chordjack is straightforward in the same sense as jumpstream, but challenges a completely different skill set which players are absurdly good at, which can lead to a single 200 causing a team to lose a point they would've won otherwise. You can be sure to look forward to some extremely high scores for this pick in future rounds.
```

Comment:
imo feels more appropriate to capitalize slot names, thoughts?
```suggestion
The mappool size has increased for this week by introducing two new slots — Chordjack ([RC3](https://osu.ppy.sh/beatmapsets/2238657#mania/4757017)) and Hybrid 3 ([HB3](https://osu.ppy.sh/beatmapsets/2238644#mania/4756987)). Chordjack is straightforward in the same sense as jumpstream, but challenges a completely different skill set which players are absurdly good at, which can lead to a single 200 causing a team to lose a point they would've won otherwise. You can be sure to look forward to some extremely high scores for this pick in future rounds.
```

Reply:
https://github.com/ppy/osu-wiki/pull/12096/commits/6340f0dbe1bd20209e54c017a5a78bf914ec8296